Homework1/task4.py

```diff
@@ -0,0 +1,12 @@
+# Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, 
+# если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+# *Пример:*
+# 3 2 4 -> yes
+# 3 2 1 -> no
+m = int(input("ширина шоколадки в дольках: "))
+n = int(input("длинна шоколадки в дольках: "))
+k = int(input("сколько долек отломим: ")) 
+if ((k % n == 0 or k % m == 0) and k < n * m):
```

Comment:
Внешняя скобка не нужна. И последнее условие можно расширить.
```suggestion
if (k % n == 0 or k % m == 0) and 0 < k < n * m:
```